### PR TITLE
Updates aide-conf docs

### PR DESCRIPTION
**Pull request by kagarmoe**
Signed-off-by: kagarmoe <kgarmoe@chef.io>

### CODE REVIEW

File: docs/resources/aide_conf.md.erb [hunk 1]
```diff
@@ -56,16 +60,17 @@ The following examples show how to use this InSpec audit resource. For a full li
...
+
+    describe aide_conf.all_have_rule('sha512') do
+      it { should eq true }
+    end
 <br>
 
 ## Matchers
 
-This InSpec audit resource has the following matchers. For a full list of available matchers please visit our [matchers page](https://www.inspec.io/docs/reference/matchers/).
+This InSpec audit resource uses the matchers `eq` and `include`.
```
Comment (adamleff) [inline docs/resources/aide_conf.md.erb]: I think this line should be removed. Our examples might use these matchers, but all the standard matchers (of which `eq` and `include` are a member) can apply to this resource. Referring to the full matcher list should be sufficient.

**Review by adamleff — DISMISSED** [review]

File: docs/resources/aide_conf.md.erb (hunk 1 above)
Comment (kagarmoe) [reply docs/resources/aide_conf.md.erb]: OK. Can we remove it everywhere?

**Review by kagarmoe — COMMENTED** [review]
Made @adamleff 's changes

**Comment by adamleff** [comment]
@kagarmoe you've got a merge conflict on this still, so you'll need to rebase on master and straighten that out. Let me know if you need assistance with that.

**Review by jquick — APPROVED** [review]

**Review by chris-rock — APPROVED** [review]
Thank you @kagarmoe

**Pull request merged**